app/ts/utils/bigint.ts

```diff
@@ -84,6 +84,13 @@ export function checksummedAddress(address: bigint) {
 	return ethers.getAddress(addressString(address))
 }
 
+export function stringToAddress(addressString: string | undefined) {
+	if (addressString === undefined) return undefined
+	const trimmedAddress = addressString.trim()
+	if (!ethers.isAddress(trimmedAddress)) return undefined
```

Comment:
Feels like this could just be a regex instead.

Reply:
I feel regex is more complex than using ethers method?

Reply:
🤷 
```ts
/0x[a-fA-F0-9]{40}/.test(trimmedAddress)
ethers.isAddress(trimmedAddress)
```
If Ethers is doing checksumming then I agree with you.

Reply:
yeah, I feel `isAddress` is more clear than regexp there. If done with regexp, I would wrap that in "isAddress" function anyway to comment it